<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imag/img/12.png);
color: rgb(49, 49, 74);
font: 75 italic 14pt &quot;Bookman Old Style&quot;;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_U"/>
    <addaction name="action_R"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
    <addaction name="action_time"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="action_font"/>
    <addaction name="action_color"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="separator"/>
    <addaction name="action_QT"/>
    <addaction name="action_web"/>
    <addaction name="action_soft"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>黄大傻专栏&amp;</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_O"/>
   <addaction name="action_S"/>
   <addaction name="action_A"/>
   <addaction name="separator"/>
   <addaction name="action_U"/>
   <addaction name="action_R"/>
   <addaction name="action_A_2"/>
   <addaction name="separator"/>
   <addaction name="action_font"/>
   <addaction name="action_color"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/new.png</normaloff>:/imag/img/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/open.png</normaloff>:/imag/img/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/save.png</normaloff>:/imag/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/resave.png</normaloff>:/imag/img/resave.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/exit.png</normaloff>:/imag/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_U">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/undo.png</normaloff>:/imag/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/redo.png</normaloff>:/imag/img/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/Cut.png</normaloff>:/imag/img/Cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/copy.png</normaloff>:/imag/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/paste.png</normaloff>:/imag/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/all.png</normaloff>:/imag/img/all.png</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="action_time">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/time.png</normaloff>:/imag/img/time.png</iconset>
   </property>
   <property name="text">
    <string>时间&amp;</string>
   </property>
  </action>
  <action name="action_font">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/font.png</normaloff>:/imag/img/font.png</iconset>
   </property>
   <property name="text">
    <string>字体&amp;</string>
   </property>
  </action>
  <action name="action_color">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/color.png</normaloff>:/imag/img/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色&amp;</string>
   </property>
  </action>
  <action name="action_H">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/help.png</normaloff>:/imag/img/help.png</iconset>
   </property>
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="action_QT">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/关于.png</normaloff>:/imag/img/关于.png</iconset>
   </property>
   <property name="text">
    <string>关于QT&amp;</string>
   </property>
  </action>
  <action name="action_web">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/官网.png</normaloff>:/imag/img/官网.png</iconset>
   </property>
   <property name="text">
    <string>关于官网&amp;</string>
   </property>
  </action>
  <action name="action_soft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/软件.png</normaloff>:/imag/img/软件.png</iconset>
   </property>
   <property name="text">
    <string>关于软件&amp;</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/c.png</normaloff>:/imag/img/c.png</iconset>
   </property>
   <property name="text">
    <string>哈哈&amp;</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/a.png</normaloff>:/imag/img/a.png</iconset>
   </property>
   <property name="text">
    <string>白痴(真的)&amp;</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/b.png</normaloff>:/imag/img/b.png</iconset>
   </property>
   <property name="text">
    <string>大胸(假的)&amp;</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/imag/img/d.png</normaloff>:/imag/img/d.png</iconset>
   </property>
   <property name="text">
    <string>以上点击无效</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
